CHANGELOG

```diff
@@ -1,3 +1,6 @@
+To be released
+    - Add background-color mixin
+    - Deprecate `base-bg-color`, `bg-color-neutral-30`, `bg-color-error`, and `bg-color-light-error` mixins
```

Comment:
I'll present an alternative that prevents backwards breaking change: instead of removing the mixins, deprecate them by changing them to use the new mixins, and also have them trigger a warning for its deprecation.

This will prevent us from having to make a major version change, and all current projects on this version will continue to work.